atd-vze/src/views/Crashes/Maps/CrashMap.js

```diff
@@ -84,7 +84,7 @@ export default class CrashMap extends Component {
         height="100%"
         mapStyle={
           isDev
-            ? "mapbox://styles/mapbox/satellite-streets-v9"
+            ? "mapbox://styles/mapbox/satellite-streets-v11"
```

Comment:
@johnclary Here's that "unrelated" version bump - does this seem good to set to 11, since that's what used in the other style definitions for the display and edit versions of the map?

Reply:
yep 100% - i think it's safe to assume we want Mapbox's latest and greatest opinions on basemap styles. glad you caught that!